code/modules/virus2/effect.dm

```diff
@@ -365,12 +365,18 @@
 /datum/disease2/effect/sneeze
 	name = "Coldingtons Effect"
 	stage = 1
-	activate(var/mob/living/carbon/mob,var/multiplier)
+	activate(var/mob/living/carbon/mob, var/multiplier)
 		if (prob(30))
 			mob << "<span class='warning'>You feel like you are about to sneeze!</span>"
-		sleep(5)
+		addtimer(CALLBACK(src, .proc/do_sneeze, mob, multiplier), 5)
+
+
+	proc/do_sneeze(mob/living/carbon/mob, multiplier)
```

Comment:
For ref, I staved off refactoring this file as Abby is working on some sort of virology overhaul. As to not break her sanity.